custom/icds_reports/models.py

```diff
@@ -15,7 +15,7 @@ class AwcLocation(models.Model):
     district_id = models.TextField()
     district_name = models.TextField(blank=True, null=True)
     district_site_code = models.TextField(blank=True, null=True)
-    state_id = models.TextField()
+    state_id = models.TextField(primary_key=True)
```

Comment:
Why did you change this primary key? I don't think that's correct

Reply:
I generated models by ```manage.py inspectdb```